app/Config/Schema/sonerezh_pgsql.php

```diff
@@ -64,7 +64,7 @@ public function after($event = array()) {
 		'path' => array('type' => 'string', 'null' => true, 'default' => null, 'length' => 1024),
 		'cover' => array('type' => 'string', 'null' => true, 'default' => null),
 		'playtime' => array('type' => 'string', 'null' => true, 'default' => null, 'length' => 9),
-		'track_number' => array('type' => 'integer', 'null' => true, 'default' => null),
+		'track_number' => array('type' => 'string', 'null' => true, 'default' => null),
```

Comment:
Will the UI display the correct order when the datatype is string?
If so, the change should be done for MySQL as well.